src/utils/chartJs/CustomTooltip.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  tooltip: {
+    background: 'rgba(0, 0, 0, .8)',
+    color: 'white',
+    borderRadius: '4px',
+    pointerEvents: 'none',
+  },
+  tooltipKey: {
+    display: 'inline-block',
+    width: '10px',
+    height: '10px',
+    marginRight: '10px',
+  },
+};
+
+function Footer({ dataset, index, currData }) {
+  const prevData = dataset[index - 1].value;
+  const prevRevision = dataset[index - 1].revision;
+  const delta = (currData - prevData).toFixed(2);
+  const deltaPercentage = ((currData / prevData - 1) * 100).toFixed(2);
+  const currRevision = dataset[index].revision;
+  const hgURL = `https://hg.mozilla.org/mozilla-central/pushloghtml?fromchange=${prevRevision}&tochange=${currRevision}`;
```

Comment:
I solved my confusion from last night:  We do not need the previous revision to show `currRevision`. Here is an example URL:

    https://hg.mozilla.org/mozilla-central/pushloghtml?changeset=d326a9d5f77b

Reply:
... so you can pull it out of the <React.Fragment>

Reply:
Yup, you are right, bravo!! :)
Should I use the short version of the hash(?)  (12 chars) to generate the link or the long one?

Reply:
Longer revision is better.